examples/connext_dds/detect_samples_dropped/README.md

```diff
@@ -0,0 +1,55 @@
+# Example Code: Print QoS
+
+Example about detecting sample dropped.
+
+## Concept
+
+RTI Connext DDS provides information through statistics about every sample that
+is not seen by a subscribing application.
+Dropped samples are exposed through counters in the DataReaderCacheStatus and
+DataReaderProtocolStatus:
+
+| Field Name | Description |
+| ---------- | ----------- |
+| old_source_timestamp_dropped_sample_count | The number of samples dropped as a result of receiving a sample older than the last one, using DDS_BY_SOURCE_TIMESTAMP_DESTINATIONORDER_QOS.|
+| tolerance_source_timestamp_dropped_sample_count |The number of samples dropped as a result of receiving a sample in the future, using DDS_BY_SOURCE_TIMESTAMP_DESTINATIONORDER_QOS. |
+| ownership_dropped_sample_count | The number of samples dropped as a result of receiving a sample from a DataWriter with a lower strength, using Exclusive Ownership. |
+| content_filter_dropped_sample_count | The number of user samples filtered by the DataReader due to Content-Filtered Topics. |
+| time_based_filter_dropped_sample_count | The number of user samples filtered by the DataReader due to DDS_TimeBasedFilterQosPolicy. |
+| expired_dropped_sample_count | The number of samples expired by the DataReader due to DDS_LifespanQosPolicy or the autopurge sample delays. |
+| virtual_duplicate_dropped_sample_count | The number of virtual duplicate samples dropped by the DataReader. A sample is a virtual duplicate if it has the same identity (Virtual Writer GUID and Virtual Sequence Number) as a previously received sample. |
+| replaced_dropped_sample_count | The number of samples replaced by the DataReader due to DDS_KEEP_LAST_HISTORY_QOS replacement. |
+| writer_removed_batch_sample_dropped_sample_count | The number of batch samples received by the DataReader that were marked as removed by the DataWriter. |
+
```

Comment:
Add a sentence: 
These statistics are also published by the monitoring libraries and can be viewed using RTI montior.